src/form/ColorMapInput.tsx

```diff
@@ -244,21 +301,41 @@ const ColorMapInput: React.FC<ColorMapProps & InjectedIntlProps> = (props) => {
           <span>{minValue}</span>
           <span>{maxValue}</span>
         </div>
-        <SelectBox
-          title={''}
-          name={name}
-          choices={colorMaps}
-          value={(colorMapType && colorMapType.colorMap) || null}
-          validated={validated}
-          errorMessage={errorMessage}
-          triedToSubmit={triedToSubmit}
-          valueChanged={colorMapChanged}
-          placeholder={placeholderColorMapSelection}
-          showSearchField={true}
-          onFocus={onFocus}
-          onBlur={onBlur}
-          readOnly={readOnly}
-        />
+        <div style={{position: "relative"}}>
+          <SelectBox
+            title={''}
+            choices={[["Custom colormap","Custom colormap","+ Create new colormap for this raster"],...colorMaps]}
+            value={JSON.stringify(colorMapValue.customColormap) !=="{}" && JSON.stringify(colorMapValue.options) ==="{}" ? "Custom colormap" : (colorMapType && colorMapType.colorMap) || null}
+            name={name + '_colorMapselect'}
```

Comment:
This will probably break the dynamic help text in the explain side column (it depends on the name of the field and this field name must be included in the `helpTextForForms.tsx` component)

Reply:
gave the colormap-select versus the min and max different id's and added those to helpTextForms.tsx